src/i18n/es.js

```diff
@@ -158,12 +158,12 @@ Encontrarás un Email con los pasos a seguir.`,
     download_msg: `Descargar`,
     downloads_column: `Descargas`,
     due_date_column: `Fecha Vencimiento`,
-    error_msg: `Hubo un error al recopilar las facturas.`,
-    no_data_msg: `No hay facturas para mostrar`,
-    no_download_msg: `No hay factura para descargar`,
+    error_msg: `¡Ouch! Ocurrió un error al intentar cargar tus facturas.`,
+    no_data_msg: `Aún no hay facturas emitidas para mostrar.`,
+    no_download_msg: `Aún no hay factura para descargar`,
     paid_to_date_column: `Pagado`,
     product_column: `Producto`,
-    sub_title: `Listado de facturas emitidas`,
+    sub_title: `Aquí está detalle de las facturas que hemos emitido. Recuerda que puedes descargar cada una.`,
```

Comment:
How about this? If I'm grasping the idea ... 
Recuerda que puedes descargar cada una. --> Recuerda que puedes descargar cada una de ellas por separado.